iFixit/react-commerce
# Upgrade strapi to `4.11.0`

Closes #1738

## QA

Smoke test https://bump-strapi-4-11-0.govinor.com/admin and Next.js pages that use Strapi.

Make sure that you can successfully run `pnpm install:all` locally.
